src/main.js

```diff
@@ -103,9 +104,12 @@ var currentPoster;
 
 // event listeners go here 👇
 
+window.addEventListener('load', function() {
```

Comment:
Possible spot for refactoring? create a function that would be called upon window load